src/main/kotlin/com/grepp/nbe1_3_team04/reservation/service/response/ReservationsResponse.kt

```diff
@@ -0,0 +1,35 @@
+package com.grepp.nbe1_3_team04.reservation.service.response
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.grepp.nbe1_3_team04.global.exception.ExceptionMessage
+import com.grepp.nbe1_3_team04.reservation.domain.ParticipantGender
+import com.grepp.nbe1_3_team04.reservation.domain.Reservation
+import java.time.LocalDateTime
+
+data class ReservationsResponse(
+    val reservationId: Long,
+    val courtId: Long,
+    val memberId: Long,
+    val teamId: Long,
+    @field:JsonFormat(
+        shape = JsonFormat.Shape.STRING,
+        pattern = "yyyy-MM-dd'T'HH:mm:ss"
+    ) @param:JsonFormat(
+        shape = JsonFormat.Shape.STRING,
+        pattern = "yyyy-MM-dd'T'HH:mm:ss"
+    ) val matchDate: LocalDateTime,
```

Comment:
```suggestion
    @field:JsonFormat(
        shape = JsonFormat.Shape.STRING,
        pattern = "yyyy-MM-dd'T'HH:mm:ss"
    ) 
    @param:JsonFormat(
        shape = JsonFormat.Shape.STRING,
        pattern = "yyyy-MM-dd'T'HH:mm:ss"
    ) 
    val matchDate: LocalDateTime,
```

들여쓰기 이렇게 하는게 더 좋을거같아여